lang/py/avro/test/gen_interop_data.py

```diff
@@ -73,6 +73,7 @@ def generate(schema_file: TextIO, output_path: IO) -> None:
             continue
         base, ext = os.path.splitext(output_path.name)
         Path(f"{base}_{codec}{ext}").write_bytes(data)
+    output_path.close()
```

Comment:
This should probably be in `finally` or `with closing()`

https://docs.python.org/3/library/contextlib.html#contextlib.closing

Reply:
Oh, sorry. I've fixed it.